### DISCUSSION TITLE
Help on setting up project on local to write some custom scripts

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#5197] [Q&A] [answered]
Hello There,

Can I please get some help on setting up the project on my local machine, so that I can write some custom scripts.

**Comment by ossdhaval** [comment]
Tagging @maduvena

**Comment by maduvena** [comment, marked as answer]
1: Install Jans server - https://docs.jans.io/v1.0.13/admin/install/vm-install/ubuntu/
2: Develope Guide - https://docs.jans.io/v1.0.13/admin/developer/interception-scripts/
